bootstrap/bootstrap.go

```diff
@@ -329,6 +351,8 @@ func (b *Bootstrap) setUp() error {
 
 // tearDown is called before the bootstrap exits, even on error
 func (b *Bootstrap) tearDown() error {
+	b.shell.Headerf("Tearing down bootstrap")
```

Comment:
This might be better a `debug` comment, or a header only if debugging is on? I think 9/10 it'll be an empty header?

Reply:
Agreed. It was debugging, will remove it entirely.